### ISSUE TITLE
Create updated version of online status facet with new Sociable status

### ISSUE THREAD

**Issue by Frooxius** [Yellow-Dog-Man/Resonite-Issues#2173]
### Describe the Underlying Issue

We're adding new "Sociable" status to Resonite: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/909

The current version of facet template for switching online status needs to include this.

### Expected Behavior

Create an updated version of this facet template which includes the Sociable status. 

- The locale key is `Profile.Status.Sociable`
- The color for this is Cyan (from platform palette) 
- Use a current build in `prerelease` channel for making this, which includes the new status in the Enum and other functionaliy
- DO NOT overwrite current version, save it with "V2" suffix

**Comment by RyuviTheViali** [comment]
Okay, I have saved a new V2 version of the facet in the templates folder ready for production - all that's missing is a new icon for the mode. 

![20240530164306_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/15852289/c2df85ed-1625-44a6-beda-de693b5195a1)

**Comment by TisFoolish** [comment]
A heart? Or a plus to further counterbalance the Busy status.

**Comment by shiftyscales** [comment]
I personally feel like "Sociable" should be above online @RyuviTheViali? That way the statuses are sorted by availability.

"Open to be social" > "Online" > "Away momentarily, will be online again soon" > "Do not disturb me" > "I'm not here".

**Comment by shiftyscales** [comment]
A plus sounds like a great idea.

**Comment by shadowpanther** [comment]
> Or a plus to further counterbalance the Busy status.

A cross-out usually rhymes with a checkmark, would be more distinct.

**Comment by Frooxius** [comment]
Yes, I think it should be above "Online" too, because it's sort of "More online than online" kind of status. Opposite of "Busy".

**Comment by RyuviTheViali** [comment]
I've reordered the items to have Sociable on top.

![20240530170016_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/15852289/0d2d11cc-8f15-4411-916f-001c5b7e5930)

**Comment by TisFoolish** [comment]
> A cross-out usually rhymes with a checkmark, would be more distinct.

Check marks don't have symmetry and all the other statuses have symmetry in some form. It'd be sad to ruin that aesthetic

**Comment by Frooxius** [comment]
We could also go with something "spiky" like a star, to give it more "energy".

**Comment by shiftyscales** [comment]
Internally, I had made the joke name suggestion of "Online+" for this status because it's "more online than online". So a plus could unironically be a good fit I think.

When you say spiky- I imagine Kiki. If we do that- we'd have to go back and change the busy icon to Bouba (kidding). x3

**Comment by shiftyscales** [comment]
But then again, Bouba is the more inviting shape- so perhaps flip them?

**Comment by Banane9** [comment]
If going for something symmetric, I'd definitely support the +, even though it's rotationally similar to the x in Busy.

**Comment by stiefeljackal** [comment]
Although a star symbol was suggested, I also want to suggest an asterisk symbol for the new status. Those symbols, the star and the asterisk, make sense because typically those symbols are used to provide additional context to something when the symbol is placed at the end of a word or a sentence.

The plus symbol is the next best thing, but that is rather similar to the 'x' symbol as some suggested.

**Comment by lxw404** [comment]
I'm adding this info from the issue that was closed above, which originally I believed was tangential to the discussion of the sociable icon:

*All* of the icons are currently ambiguous as to which one is active when you select them.  This becomes extremely noticeable if you resize the facet to its minimal state with just the icons visible:
![Resonite_T2Wdc19sKV](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/52231149/a2ec845e-7d61-4676-9def-8b06cd8cd90c)

Guess which one of them is active above before revealing the answer below:

<details> <summary>Spoiler</summary>
<i>It was the Online (green) one</i>
</details>

I mention in that issue that the current icons do not have a consistent way of telling when they are active versus inactive.  For example, the offline icon when active looks like the online/sociable buttons + vice versa, and the outlining on the away button looks like the other icons with outlines in different states.  There should probably be a way to at a glance determine which is active without ambiguity (only one of the icons with a specific property, eg only one with an outline etc.).

**Comment by shiftyscales** [comment]
I actually knew it was online if you can believe that, @lxw404 - but I could see how someone less familiar to the platform might not recognize that.

The online status there is slightly wider to the rest because it has an outline where all of the rest don't as you suggested as a possible solution- so clearly a bit more is required than that.

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/859c3b40-72f9-4ce8-a369-350157922f3b)

E.g. maybe further thickening the outline, or inverting the visual so that all of the unselected statuses are an empty circle, and the active status is a filled (or dotted) circle similar to radio buttons.

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/1cb16795-a34d-4349-8d92-d03358c351e5)

**Comment by shiftyscales** [comment]
30 second MS paint mockup:
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/d44b71b5-d4c2-4a3b-a26c-ca146c2cb977)

**Comment by JackTheFoxOtter** [comment]
Since this is currently being overworked, I would kindly ask to take a look at the scaling / size of this facet. The space of the facet taken up (green outline on the grid when you grab it) and the actual visual size of the facet are different, which makes this one a very frequent pain point when trying to arrange my dashboard layout.

**Comment by djsime1** [comment]
Might as well chip in my two cents here,

I used an [online color-vision deficiency](https://www.color-blindness.com/coblis-color-blindness-simulator/) simulator to contrast how the Online and Sociable status look to somebody with color blindness. I figured since both Hero Green and Hero Blue have a similar hue, saturation, and luminosity, they could be difficult to distinguish for those with degraded blue perception. Sure enough, the two colors are rather difficult, if not nearly impossible to distinguish without accompanying labels. Below are the simulated results:

### Normal vision
![StatusColorContrastNormal](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/7483e76c-2214-48e8-bc2c-70cec520eb12)

### Blue-Weak/Tritanomaly: Tough to distinguish
![StatusColorContrastTritanomaly](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/23686140-cea5-4981-9770-6aa401ab1783)

### Blue-Blind/Tritanopia: Nearly identical
![StatusColorContrastTritanopia](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/7e47ff54-15f7-4203-b7e8-b2462cb5753f)

### Monochromacy/Achromatopsia: Indistinguishable
![StatusColorContrastAchromatopsia](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/34515928-7cc0-4a21-a70b-7585219c7fb1)

So, I agree that an icon is needed to distinguish the two statuses, like Away, Busy, and Invisible. I mocked up some prototype icons, and like several others in this thread, I think that a 4-point star is probably the best option in terms of distinguishability, symbolism, and simplicity. 

### Stars
![SociableStars](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/c2dd6a63-d983-4aa3-bbdf-670071fdda2f)
As I said before, these are the simplest. Easy to recognize, apart from the 5-point star, which might be interpreted as a "favorite" or otherwise special contact by a new user. These should also be readable at a small scale.

### Dots
![SociableDots](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/c022be1c-b6fb-4bc3-b143-3254de57bb85)
I guess these could be thought of as representing a conversation circle, but I don't think they offer enough visibility, especially at smaller scales.

### Symbols
![SociableSymbols](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/a6c4709b-27a6-4c6b-962f-504144a934ea)
These intentionally convey the point, but when compared to the other status icons they are too noisy. They could also be mistaken for meaning you have a new message, or you're being invited. (Comment icon from Nucleo Core icons, Waving Hand modified from Twemoji)

On a minor side note, as brought up by lxw404 earlier in the thread, having your current status be Online makes it the same shape as Invisible. I've not tested the contrast through colorblindess filters, but the lower luminosity of Hero Purple should make it different enough from Hero Green. Still, another point of accessability worth considering.

**Comment by Frooxius** [comment]
We definitely should go for a symbol and get one made for this. I don't think using only color is a good idea, especially for the color blindness. 

The text helps distinguish them right now, but having more distinguishing factors helps overall and some people seem to like to create a condensed version.
